pyoozie/oozie_api.py

```diff
@@ -0,0 +1,561 @@
+from __future__ import unicode_literals
```

Comment:
As an aside, this file is really long, but I don't think we should split it up. The linter is wrong in this case.